src/User/User.php

```diff
@@ -205,6 +261,18 @@ public function getId(): ?string
         return $this->getIdentity()->getId();
     }
 
+    /**
+     * Set the name of the cookie identity
+     * @param string $name New name of the cookie
+     * @return this
+     */
+    public function setIdentityCookie(string $name): this
```

Comment:
Undefined class 'this' 

self?

Reply:
ahahaha is true. I updated in my last commit :)
Thanks!!